src/screenshot.cpp

```diff
@@ -831,6 +832,40 @@ bool MakeHeightmapScreenshot(const char *filename)
 	return sf->proc(filename, HeightmapCallback, nullptr, MapSizeX(), MapSizeY(), 8, palette);
 }
 
+static ScreenshotType _screenshot_type;
```

Comment:
This variable has lost its "confirmed" name, and its docstring

Reply:
There were two different variables (in two different files) serving the exact same purpose. In retrospect, I probably could have saved myself a lot of trouble by using the implementation in `toolbar_gui.cpp` as the canonical reference.